stockflux-securities-master/src/components/TableBody.js

```diff
@@ -0,0 +1,59 @@
+import React from "react";
```

Comment:
Hmm... I guess I meant you would move it out into a jsx (not a separate component) like this:

```
const tableBody  () => [ return jsx here]
```

Let's see what @jrhenderson1988 thinks as it works both ways.

Reply:
Alright, done now